common/button/button-fr.html

```diff
@@ -0,0 +1,428 @@
+---
+{
+	"title": "Boutons",
+	"language": "fr",
+	"altLangPage": "button-en.html",
+	"description": "Page des boutons comprenant des exemples de travail pour tester l'apparence et le fonctionnement de divers styles de boutons.",
+	"dateModified": "2023-06-14"
+}
+---
+
+<h2>Sur cette page:</h2>
+
+<ul>
+	<li><a href="#purpose">Usage</a></li>
+	<li><a href="#basic">Utilisation de base</a></li>
+	<li><a href="#enhanced">Utilisation améliorée</a></li>
+	<li><a href="#colours/styles">Couleurs/Styles</a></li>
+	<li><a href="#calltoaction">Appel à l'action</a></li>
+	<li><a href="#sizing">Taille</a></li>
+	<li><a href="#fullwidth">Largeur maximale</a></li>
+	<li><a href="#activestate">État d'activation</a></li>
+	<li><a href="#disabledstate">État de désactivation</a></li>
+	<li><a href="#bghorizontal">Groupes de Bouton (Horizontale)</a></li>
+	<li><a href="#bgvertical">Groupes de Bouton (Verticale)</a></li>
+	<li><a href="#buttontoolbar">Barres à outils</a></li>
+	<li><a href="#inputgroups">Groupes d'entrée</a></li>
+	<li><a href="#datatoggle">Boutons bascule de données</a></li>
+</ul>
+
+<h2>Bouton contre liens</h2>
+<dl class="dl-horizontal">
+<dt>Bouton</dt>
+<dd>Un bouton effectuera une action sur la page actuelle sans déplacer le focus ou sans éloigner l'utilisateur</dd>
+<dt>Lien</dt>
+<dd>Un lien enverra l'utilisateur à un emplacement, dans la même page via un lien d'ancrage (`#`) ou vers une autre page sans envoyer de données.</dd>
+</dl>
+<p>N'utilisez pas le bouton de rôle sur une ancre, utilisez plutôt un élément de bouton.</p>
+
+<h2 id="basic">Utilisation de base</h2>
+
+<h3>Par défaut</h3>
```

Comment:
There is no h3 sibling under the h2 scope, Please merge this title with the h2 parent and bump up the all the sub heading.

```suggestion
<h2 id="basic">Utilisation de base (Par défaut)</h2>
```

Reply:
Changes made in the commit "Updated working examples".